docs/source/plugins/for_plugin_developers.rst

```diff
@@ -243,6 +243,15 @@ functionality by simply installing your package along with napari:
 
    pip install napari mypackage
 
+Step 4: Share your plugin with the world
+----------------------------------------
+
+If you are using Github, add the `"napari-plugin" topic
+<https://github.com/topics/napari-plugin>`_ to your repo so other developers can
+see your work.
+
+When you are ready for users, announce your plugin on the `Image.sc Forum
+<https://forum.image.sc/tag/napari>`_.
```

Comment:
can we also add this step to the overview at the top?  It's a little confusing, since it has 4 steps in the overview, but only three in the body... so feel free to edit that as needed (e.g. make one 3a/3b).

In this section about sharing with the world, I think we should also add something about publishing to PyPI, since that's ultimately where people will be installing from (for instance, in #1357).  The cookiecutter has [detailed information on how to publish](https://github.com/napari/cookiecutter-napari-plugin#set-up-automatic-deployments), and we should probably add some of that here too.

Reply:
Ah, gotcha. Missed that. Will do.

Reply:
since a lot of that cookiecutter info assumes they are actually using the cookiecutter, it's fine if we just want to say something vague here about publishing to pypi, with perhaps a link to the cookiecutter docs "if you're using the cookiecutter", and some more general pypi docs otherwise?  For the purpose of this PR, I think just a sentence would be great

Reply:
Just saw this comment and that's *exactly* what I just did. We've got a mind-meld happening here, @tlambert03.